backend/api/chat.py

```
@@ -110,341 +114,44 @@ async def chat(
-            # 新規スレッド: 初回ユーザーメッセージから作成
-            first_user_message = next(
-                (msg for msg in request.messages if msg.role == "user"), None
-            )
-            if first_user_message is None:
-                raise HTTPException(
-                    status_code=400,
-                    detail="At least one user message is required for new thread",
-                )
+        get_model(model_name)
+    except ValueError as e:
+        logger.error("Invalid model name: %s", model_name)
```

Ruff (0.14.11)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)